lib/shopify_api/rest/request.ex

```diff
@@ -21,8 +21,8 @@ defmodule ShopifyAPI.REST.Request do
 
   ## Public Interface
 
-  def perform(%AuthToken{} = token, method, path, body \\ "") do
-    url = url(token, path)
+  def perform(%AuthToken{} = token, method, path, body \\ "", params \\ []) do
+    url = add_params_to_url(url(token, path), params)
```

Comment:
:hammer_and_wrench: this might make more sense if pipelined:

```suggestion
    url = token |> url(path) |> add_params_to_url(params)
```